ylsbrain/journal/2026-06-14.md

```
@@ -0,0 +1,55 @@
+# yls brain — journal 2026-06-14
+
+### [00:00] /fewer-permission-prompts — read-only allowlist analysis (apply BLOCKED by auto-mode)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)